repos/spack_repo/builtin/packages/openmpi/package.py

```diff
@@ -780,6 +780,8 @@ def patch(self):
     conflicts("~internal-pmix", "+internal-hwloc")
     conflicts("~internal-pmix", "+internal-libevent")
 
+    conflicts("%fortran=clang")
```

Comment:
Similar story here: confirmed to get passed the configure step with LLVM 20 (`flang` == `flang-new`). For LLVM 19 and 18 you need to have `FC=flang-new`.